gems/pending/spec/miq_vm/miq_azure_vm_instance_spec.rb

```diff
@@ -74,7 +74,7 @@
       let(:expected_num_roots)    { 1 }
       let(:expected_guest_os)     { "Linux" }
       let(:expected_num_fs)       { 1 }
-      let(:expected_num_fs_types) { ["Ext4"] }
+      let(:expected_num_fs_types) { ["XFS"] }
```

Comment:
@bronaghs The test config uses `miq-test-ubuntu1` not `miq-test-rhel1`, that's probably why the fs type changed.